EIPS/eip-draft_difficulty_bomb_delay.md

```diff
@@ -0,0 +1,62 @@
+---
+eip: <to be assigned>
+title: Delaying Difficulty Bomb to Mid August 2022
+description: Delays the difficulty bomb by a further 500000 blocks, to the middle of August 2022.
+author: Tomasz Kajetan Stanczak (@tkstanczak), Eric Marti Haynes (@ericmartihaynes), Josh Klopfenstein (@joshklop)
+discussions-to: https://ethresear.ch/t/delaying-difficulty-bomb-to-mid-august-2022/12762
+status: Draft
+type: Standards Track
+category: Core
+created: 2022-06-01
+---
+
+*Note*: this EIP mirrors [EIP-4345](eip-4345.md) (which itself follows [EIP-649](eip-649.md), [EIP-1234](eip-1234.md), [EIP-2384](eip-2384.md) and [EIP-3554](eip-3554.md)), from which much of the text and code is taken.
+
```

Comment:
```suggestion
```
Not a valid EIP section.  Also, unnecessary and can simply be mentioned in the discussions-to link.